src/client.ts

```diff
@@ -458,6 +459,70 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
     }
   }
 
+    /**
+   * Update the default resolution of the selected video track for the sdk.
+   *  Pass in the following:
+   *  - resolution: Either undefined or an object containing ConstrainULongs
+   *      representing both width and height
+   *  - updateActiveSessions: boolean
+   *
+   * If `updateActiveSessions` is `true`, any active sessions will
+   *  have their video resolutions updated to either the selected resolution
+   *  or the next best resolution the device is capable of
+   *
+   * Else, only the sdk defaults will be updated and active sessions'
+   * resolutions will not be touched
+   *
+   * Success or failure of `applyConstraints` will emit an event for
+   * the consuming app to listen for so that it can determine if the
+   * resolution they requested is actually in place or the next best
+   * option. This will allow the consuming app to notify the user as
+   * they see fit.
+   *
+   * @param resolution the width and height that was requested
+   * to update the video; could be undefined to represent the default value
+   *
+   * @returns a promise that fullfils once the event has been emitted
+   * signaling that the resolution has updated
+   */
+  async updateDefaultResolution(resolution: IVideoResolution | undefined, updateActiveSessions: boolean): Promise<any> {
+    this._config.defaults.videoResolution = resolution;
+    if (updateActiveSessions) {
```

Comment:
NIT: rather than wrapping all the logic of the rest of this function inside an if, it looks a little cleaner to just do: 
``` ts
if (!updateActiveSessions) {
  return;
}
```
Keep the code cleaner IMO – especially when you have lots of other nested brackets throughout.